test/test-grpc-context.js

```diff
@@ -60,7 +61,7 @@ Object.keys(versions).forEach(function(version) {
 
       common.replaceDebugLogger(agent, function(error, uri) {
         if (error.indexOf('http') !== -1) {
-          assert.notStrictEqual(uri.indexOf('localhost'), -1);
+          httpDebugPrinted = true;
```

Comment:
The test was broken before (in a similar manner to `test-plugins-sample-warning.js`) since the condition `error.indexOf('http') !== -1` was never true. The old assert checks the format of the message being printed in the `http` hook; the format has since changed. Asserting in `afterEach` rather than here just makes sure we don't (erroneously) fail if additional debug-level messages are printed.